pkgs/applications/networking/cluster/openshift/default.nix

```diff
@@ -71,16 +74,12 @@ in stdenv.mkDerivation rec {
   '';
 
   installPhase = ''
-    mkdir -p "$out/bin"
-    cp -a "_output/local/bin/$(go env GOOS)/$(go env GOARCH)/"* "$out/bin/"
+    mkdir -p $bin/bin $out
+    cp -a "_output/local/bin/$(go env GOOS)/$(go env GOARCH)/"* "$bin/bin/"
     install -D -t "$out/etc/bash_completion.d" contrib/completions/bash/*
     install -D -t "$out/share/zsh/site-functions" contrib/completions/zsh/*
```

Comment:
I would also put that in `$bin` so "${openshift}//share/zsh/site-functions" works as expected.

Reply:
ok :+1: